homeassistant/components/shelly/climate.py

```diff
@@ -316,5 +316,6 @@ def _handle_coordinator_update(self) -> None:
                 self.block = block
 
                 _LOGGER.debug("Entity %s attached to block", self.name)
```

Comment:
```suggestion
                _LOGGER.debug("Entity %s attached to blocks", self.name)
```

and move it in the if below

Simone